src/tracers.jl

```diff
@@ -12,147 +12,142 @@ The provided index set type `S` has to satisfy the following conditions:
 Subtypes of `AbstractSet{<:Integer}` are a natural choice, like `BitSet` or `Set{UInt64}`.
 """
 
+empty_sparse_vector(T) = T()
+empty_sparse_matrix(T) = T()
+
+sparse_vector(T, index) = T([index])
+sparse_vector(::Type{T}, index) where {T<:DuplicateVector} = T(index)
+sparse_vector(::Type{T}, index) where {T<:SortedVector} = T(index)
+
+#================#
+# Elementwise OR #
+#================#
+
+## We use ∨ to represent the elementwise OR
+# TODO REVIEW: \vee looks too much like the character 'v' in code
```

Comment:
You're right. Let's spell out the operators then:

- `elementwise_or`
- `outer_product_or`
- `symmetric_outer_product_or` (not essential)

Reply:
on the other hand this makes a lot of code more verbose, and further from the paper

syntax highlighting treats `\vee` differently from `vee` in Julia, so maybe we're good?

in which case you would use `\vee^T` for the outer product maybe

```julia
∨  # vee
v  # v
```

Reply:
Good news: there is also `\bigvee`, which looks a tad more distinguishable:
```julia
⋁ # bigvee
∨ # vee
v # v
V # V
```
What happened to the idea of using `\oplus` ⊕ for the element-wise OR/max?

Reply:
If we switch to abstract sets everywhere, vee becomes a union so the worry disappears.

As for the oplus, since we're only using booleans, a max-plus analogy was overkill. Max for booleans is just OR